example/all.yaml

```diff
@@ -59,6 +59,7 @@ linter:
     - no_adjacent_strings_in_list
     - no_duplicate_case_values
     - non_constant_identifier_names
+    - null_closures
```

Comment:
This name doesn't seem consistent, but @pq can probably comment on that better than I.

Reply:
Unfortunately, we're not hugely self-consistent.  (Try as I did to keep it that way.)

It might be consistent w/ more lints if it were more prescriptive (`no`... or `avoid`...) but at this point I'm sort of wishing we'd never added those since they may get out of sync with content.  For example, if we want to promote avoid advice to no, it's awkward if the lint name has avoid baked in...

How's that for a non answer!

Reply:
Yeah I'm happy to rename or leave it.  I think i named it after something like `empty_statements` and `invariant_booleans`.

Also, I would not be against a wide-reaching rename on all the lints after establishing a policy, and providing aliases after the fact. (So after renames, `empty_statements` would be an alias for `avoid_empty_statements`.